misc/userscripts/qute-keepass

```diff
@@ -148,7 +148,7 @@ def dmenu(items, invocation, encoding):
 
 def get_password():
     """Get a keepass db password from user."""
-    _app = QApplication(sys.argv)
+    _app = QApplication(sys.argv) # don't remove this local variable
```

Comment:
Not sure if we lint these userscripts but there is some PEP that says there needs to be two spaces before comments on the same line as code.

Reply:
Np, done.